jobs/e2e-test.groovy

```diff
@@ -489,13 +491,13 @@ def runLambda(){
              string(name: 'REVISION_DESCRIPTION', value: REVISION_DESCRIPTION),
              string(name: 'DEPLOYER_USERNAME', value: params.DEPLOYER_USERNAME),
              string(name: 'URL', value: E2E_URL),
-             string(name: 'NUM_WORKER_MACHINES', value: params.NUM_WORKER_MACHINES),
+             string(name: 'NUM_WORKER_MACHINES', value: "30"),
```

Comment:
**note:** Hardcoding this value here so it doesn't conflict with the Python/selenium workers. This number is used separately so LambdaTest uses it for how many machines we need to use in their platform.

Reply:
Makes sense.  I think this will be clearer once you rename NUM_WORKER_MACHINES in e2e-test-cypress to have a different name.

Reply:
(But maybe just don't pass in this value at all, and let the cypress job use its default?)

Reply:
yeah I like that, I'm going to get rid of this call here.